numba/core/extending.py

```diff
@@ -54,7 +54,8 @@ def generic(self):
 _overload_default_jit_options = {'no_cpython_wrapper': True}
 
 
-def overload(func, jit_options={}, strict=True, inline='never'):
+def overload(func, jit_options={}, strict=True, inline='never',
+             signatures=None):
```

Comment:
The signatures arguments can be removed for this PR.

Reply:
done